sdk/devcenter/azure-developer-devcenter/swagger/README.md

```diff
@@ -0,0 +1,26 @@
+### Settings
+
+```yaml
+input-file:
+  - C:\Users\chrismiller\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\devcenter.json
+  - C:\Users\chrismiller\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\devbox.json
+  - C:\Users\chrismiller\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\environments.json
```

Comment:
No local path, can be some paths like `https://raw.githubusercontent.com/Azure/azure-rest-api-specs/<real-commit-id>/specification/devcenter/data-plane/Microsoft.DevCenter/preview...`